packages/tools/src/tools/displayTools/Labelmap/labelmapDisplay.ts

```diff
@@ -134,6 +135,51 @@ function removeSegmentationRepresentation(
   }
 }
 
+/**
+ * Checks if a segmentation data have the same frameOfReference as the series
+ * displayed in a given viewport
+ * @param viewport
+ * @param referencedVolumeId volume id of the segmentation reference series
+ * @returns
+ */
+function isSameFrameOfReference(viewport, referencedVolumeId) {
+  // if the referencedVolumeId is not defined, we acted as before to not break
+  // applications as referencedVolumeId is inserted in this change
+  // Can modify that in the future commits
+  if (referencedVolumeId) {
```

Comment:
we should add `FrameOfReferenceUID` to the IImageVolume, and just grab it here

Reply:
ok